InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -263,9 +267,9 @@ def test_epoch_end(self, outputs: List[Dict[str, Any]]) -> None:  # type: ignore
             list_slide_dicts.append(slide_dict)
             list_encoded_features.append(results[ResultsKey.IMAGE][slide_idx])
 
-        print(f"Metrics results will be output to {fixed_paths.repository_root_directory()}/outputs")
-        csv_filename = fixed_paths.repository_root_directory() / Path('outputs/test_output.csv')
-        encoded_features_filename = fixed_paths.repository_root_directory() / Path('outputs/test_encoded_features.pickle')
+        print(f"Metrics results will be output to {fixed_paths.repository_parent_directory()}/outputs")
+        csv_filename = fixed_paths.repository_parent_directory() / Path('outputs/test_output.csv')
```

Comment:
Easier to maintain if you don't duplicate the path creation logic (first construct csv_filename, and then print off that)

Reply:
Implemented all the suggestions